.github/workflows/build.yml

```diff
@@ -81,11 +81,31 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - name: Install package
-        uses: ./.github/actions/install-cpu
+        uses: ./.github/actions/install
         with:
           editable: true
```

Comment:
Should the other with statements also have the hardware specification? i.e.
```
with:
          editable: true
          hardware: "cpu"
```

Reply:
The default choice for `hardware` is `"cpu"`; see [here](https://github.com/graphnet-team/graphnet/blob/7124c1090f12f8296f26962a043f411b6e88360a/.github/actions/install/action.yml#L20). So, it would be perfectly possible to specify `hardware: "cpu"` as you mention above, but since there is no _need_ to, and since we will only choose a setting other than `"cpu"` on very rare occasions, I think it's fine to omit.

Reply:
Personally, I like to have it for transparency, however, both work and I missed the default when reviewing. I'll consider it resolved :D